### PR TITLE
add ability to import answers and regions

### PR DESCRIPTION
**Pull request by mioi**
adding the import functionality for answers and regions within an ns1 record.

### CODE REVIEW

File: ns1/resource_answer.go [hunk 1]
```diff
@@ -268,3 +273,16 @@ func AnswerUpdate(resourceData *schema.ResourceData, meta interface{}) error {
 	}
 	return answerToResourceData(resourceData, a)
 }
+
+
+func AnswerStateFunc(resourceData *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
```
Comment (pdbogen) [inline ns1/resource_answer.go:278]: hm, is the idea really to _modify_ the `*schema.ResourceData` and then return it, or should we be allocating a new one?

**Comment by pdbogen** [comment]
Assuming the answer to my comment is "it's doing exactly what it should be doing," this LGTM.

File: ns1/resource_answer.go (hunk 1 above)
Comment (mioi) [reply ns1/resource_answer.go:278]: That's a good question. I used the function for records as an example:
https://github.com/terraform-providers/terraform-provider-ns1/blob/master/ns1/resource_record.go#L409

here are a couple other examples from different providers:
https://github.com/terraform-providers/terraform-provider-digitalocean/blob/master/digitalocean/resource_digitalocean_record.go#L206
https://github.com/terraform-providers/terraform-provider-google/blob/master/google/resource_pubsub_topic.go#L109

I can't find much documentation on the internals of what Terraform expects.

Comment (pdbogen) [reply ns1/resource_answer.go:278]: I guess as long as we're making the same mistakes other people are making (if, indeed, they're mistakes :joy:) then we're in good shape.

Comment (mioi) [reply ns1/resource_answer.go:278]: In normal circumstances (where there is a direct mapping between ID to resource, i.e. `terraform import aws_instance.instance i-0dacbedef48cf1e23`) this function is usually used:
https://github.com/hashicorp/terraform/blob/master/helper/schema/resource_importer.go#L50 which just seems to return what is inputted into the function.

In special cases, for instance to import an answer (`terraform import ns1_answer.foobar <record>/<answer>`), the `schema.ResourceData` object needs to be modified (or maybe a new one can be allocated? I'm not sure) and then returned.

**Pull request merged**